src/Explorer/Tabs/DocumentsTabV2/DocumentsTableComponent.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { TableRowId } from "@fluentui/react-components";
+import { ReactWrapper, mount } from "enzyme";
+import React from "react";
+import { DocumentsTableComponent, IDocumentsTableComponentProps } from "./DocumentsTableComponent";
+
+const PARTITION_KEY_HEADER = "partitionKey";
+const ID_HEADER = "id";
+
+describe("DocumentsTableComponent", () => {
+  const createMockProps = (): IDocumentsTableComponentProps => ({
+    items: [
+      { [ID_HEADER]: "1", [PARTITION_KEY_HEADER]: "pk1" },
+      { [ID_HEADER]: "2", [PARTITION_KEY_HEADER]: "pk2" },
+      { [ID_HEADER]: "3", [PARTITION_KEY_HEADER]: "pk3" },
+    ],
+    onItemClicked: (index: number): void => {
+      index;
```

Comment:
What's the purpose of this function? It seems to be a no-op since you're not returning the `index` value. If so, it might be clearer as `() => {}` or `(_: number): void => {}` (if the parameter/return types are essential)